Add Resources to Cloudflare Terraform Provider (#2633)
DataDog/terraform-provider-datadog: docs/resources/integration_cloudflare_account.md

### Optional

- `email` (String) The email associated with the Cloudflare account. If an API key is provided (and not a token), this field is also required.

> - `resources` (List of String) An allowlist of resources to restrict pulling metrics for including 'web', 'dns', 'lb' (load balancer), 'worker')

Reviewer: Are `web`, `dns`, etc. tags on the metrics? Or something else?

```suggestion
- `resources` (List of String) An allowlist of resources to restrict pulling metrics for including `web`, `dns`, `lb` (load balancer), 'worker')
```

Author: These are enabling which resources to collect when provisioning a cloudflare account via terraform.

Author: These docs are also autogenerated

> - `resources` (List of String) An allowlist of resources to restrict pulling metrics for including `web`, `dns`, `lb` (load balancer), `worker`)

Reviewer: ```suggestion
- `resources` (List of String) An allowlist of resources, such as `web`, `dns`, `lb` (load balancer), `worker`, to pull metrics for.
```

Author: I think the restriction part is important here, by default, they're set to all, but if they want one turned off, then they use the resources list. Phrasing as an allow list makes it seem like if they specify one, they'll collect metrics for it, when it's the opposite.

Reviewer: @apiazza-dd Ah okay, thanks for the clarification. I updated the suggestion, what do you think?

Reviewer: @apiazza-dd actually rereading your comment, I might have misunderstood.

> but if they want one turned off, then they use the resources list. 

Is the allowlist of resources, the resources that users want to pull metrics for or the ones they don't want pulling metrics for?

Author: @maycmlee These docs are autogenerated and have been updated to match our other repository. Does this look okay, or should I make a change in the other repo?

Reviewer: DM'd. Docs are autogenerated so the suggestion will be updated in the other repo.

Author: https://github.com/DataDog/datadog-api-spec/pull/3270/files
